releases/evm/2025-01-ITS-v2.1.0.md

```diff
@@ -81,11 +81,26 @@ node evm/deploy-its.js --upgrade
 
 Migration Steps
 
-1. In advance, the partner grants minter role to the corresponding token manager on each chain
-2. ITS is upgraded
-3. The partner revokes minter role from the main ITS contract
+First the token manager information needs to be collected. For mainnet, this has been done, so this step can be skipped. For testnet first go to [this](../../axelar-chains-config/info/tokenManagers-testnet.json) and create a new entry in the json object with the chain info as follows.
+```json
+"chain_name": { 
+    "start": blockNumber, 
+    "end": blockNumber, 
+    "tokenManagers": [] 
+}
+```
+For `blocknumber` go to the explorer and find the block when the its was deployed on that chain (There is sadly no way to query ethers for this information to my knowledge). Then do the following and wait for the data to be collected.
+
+```bash
+# Get all token managers on chains.
+node evm/get-token-manager-deployments.js
+```
 
-We’d also like to migrate minter for all native ITS tokens if they weren’t auto migrated, so we can remove the migration logic in the future. We’ll create scripts to do that in the future since this is not a blocker for the upgrade.
+Once you have the data collected for the chains you wish to migrate, simply run, to migrate all the tokens.
+```bash
```

Comment:
How do we migrate in batches? E.g. migrate all known tokens first, and then collect and migrate again right before the upgrade? does it amount to running these cmds again? Can we provide a time range or so so it doesn't spend time going over all historical data again?